### DISCUSSION TITLE
Congratulations on your successful project!

### DISCUSSION THREAD

**Discussion by wywywywy** [zero01101/openOutpaint#126] [General]
Looking at the replies on Reddit, the feedback seems really positive overall.

There are of course still lots to do to make it truly world-class, but I think we can confidently say it's been a success.

Well done to all the devs!

**Comment by zero01101** [comment]
lol well a ton of the momentum seems to have been due to the webUI extension, so thanks to you as well for suggesting it :)

> truly world-class

lmao am i the only person left who thinks of this whole debacle as a silly little fun goofy project to distract me from reality or what

**Reply by seijihariki** [reply]
Haha! We can consider this as a fun side project while aiming high, no problems!

And yes, the webui extension definitely expanded it's reach a lot
